src/dht.js

```diff
@@ -15,54 +11,6 @@ module.exports = (node, DHT, config) => {
   })
 
   return {
-    /**
-     * Store the given key/value pair in the DHT.
-     * @param {Buffer} key
-     * @param {Buffer} value
-     * @param {Object} [options] - put options
-     * @param {number} [options.minPeers] - minimum number of peers required to successfully put
-     * @returns {Promise<void>}
-     */
-    put: (key, value, options) => {
-      if (!node.isStarted() || !dht.isStarted) {
-        throw errCode(new Error(messages.NOT_STARTED_YET), codes.DHT_NOT_STARTED)
-      }
-
-      return dht.put(key, value, options)
-    },
-
-    /**
-     * Get the value to the given key.
-     * Times out after 1 minute by default.
-     * @param {Buffer} key
-     * @param {Object} [options] - get options
-     * @param {number} [options.timeout] - optional timeout (default: 60000)
-     * @returns {Promise<{from: PeerId, val: Buffer}>}
-     */
-    get: (key, options) => {
-      if (!node.isStarted() || !dht.isStarted) {
-        throw errCode(new Error(messages.NOT_STARTED_YET), codes.DHT_NOT_STARTED)
-      }
-
-      return dht.get(key, options)
-    },
-
-    /**
-     * Get the `n` values to the given key without sorting.
-     * @param {Buffer} key
-     * @param {number} nVals
-     * @param {Object} [options] - get options
-     * @param {number} [options.timeout] - optional timeout (default: 60000)
-     * @returns {Promise<Array<{from: PeerId, val: Buffer}>>}
-     */
-    getMany: (key, nVals, options) => {
-      if (!node.isStarted() || !dht.isStarted) {
-        throw errCode(new Error(messages.NOT_STARTED_YET), codes.DHT_NOT_STARTED)
-      }
-
-      return dht.getMany(key, nVals, options)
-    },
-
     _dht: dht,
 
     start: () => dht.start(),
```

Comment:
Is it worth having this file anymore? It's not doing much and is just adding complexity (libp2p._dht._dht instead of libp2p._dht). We could just have `libp2p._dht` be the actual DHT implementation.

Reply:
you are right, I will remove it